src/utils/deploy/hash-fns.mjs

```diff
@@ -110,7 +117,12 @@ const hashFns = async (
     assetType: 'function',
     normalizedPath: path.basename(functionPath, path.extname(functionPath)),
     runtime,
+    displayName,
   }))
+  const fnConfig = functionZips.reduce(
+    (funcs, curr) => (curr.displayName ? { ...funcs, [curr.name]: { display_name: curr.displayName } } : funcs),
+    {},
+  )
```

Comment:
this reads very complicated to me. would this have the same effect?
```suggestion
  const fnConfig = Object.fromEntries(
     functionZips
       .filter(func => !!func.displayName)
       .map(func => [func.name, { display_name: func.displayName}])
   )
```

Reply:
It would not be exactly the same effect, as the result of the reduce would be an object, and the map would be an array. But I did add a filter like you did now, in front of the reduce, maybe that already makes it a bit easier to read?

Reply:
> and the map would be an array.

that's what the Object.fromEntries would do, wouldn't it?

Reply:
Oooh sorry I totally forgot that 😆 in that case I think I would still prefer it as a reduce, for me that is a bit more readable, as it shows how the object will look. But I do understand that it might be personal preference as well 🤔

Reply:
sounds good 👍